### PR TITLE
Eslint example code.

### LINKED ISSUE #156: Should code under examples be ignored on eslinting
I am Johnson , a react fan and web developer.

**Proposal/Feature-request:**
examples code should be lint as well,  if a designer need help, some editor-based plugins could help make eslint pass, and Some contributors could also review and improve the code quality.

**Expected behavior:**
All JS code well lint and looks styled well. 

I can provide a pr, make the examples not ignored and exist examples be well lint.

also mentioned on [#58](https://github.com/airbnb/react-sketchapp/issues/58)

### PR DESCRIPTION
**Pull request by sjy**
- Resolve eslint json lint waring;
- Resolve eslint errors among examples;
- Update some examples' readme 

Fixes #156.

### CODE REVIEW

**Comment by sjy** [comment]
Any Clues ? The lint job passed on my local env. Seems the examples code is cached or refused to be override.

File: .eslintrc [hunk 1]
```diff
@@ -15,15 +15,15 @@
     "NSDate": true,
     "NSDataBase64EncodingEndLineWithCarriageReturn": true,
     "NSData": true,
-    "NSMakeRange": true,
+    "NSMakeRange": true
```
Comment (ljharb) [inline .eslintrc]: trailing commas are fine in .eslintrcs, there's no need to remove them.

File: examples/basic-setup/src/my-command.js [hunk 2]
```diff
@@ -12,7 +16,7 @@ const textColor = (hex) => {
   return chroma(hex).darken(3).hex();
 };
 
-const Swatch = ({ name, hex }) => (
+const Swatch = ({ name, hex }) =>
```
Comment (ljharb) [inline examples/basic-setup/src/my-command.js]: this isn't a good change; multiline jsx MUST always be wrapped in parens.

File: examples/basic-setup/src/my-command.js [hunk 3]
```diff
@@ -3,7 +3,11 @@ import PropTypes from 'prop-types';
 import { render, Artboard, Text, View } from 'react-sketchapp';
 import chroma from 'chroma-js';
 
-// take a hex and give us a nice text color to put over it
+/**
+ * take a hex and give us a nice text color to put over it
+ * @param {string} hex
```
Comment (ljharb) [inline examples/basic-setup/src/my-command.js]: we don't use jsdoc at airbnb; let's not add it here.

File: .eslintrc (hunk 1 above)
Comment (sjy) [inline .eslintrc]: Resolved

File: examples/basic-setup/src/my-command.js (hunk 3 above)
Comment (sjy) [inline examples/basic-setup/src/my-command.js]: Resolved;

File: examples/basic-setup/src/my-command.js (hunk 2 above)
Comment (sjy) [inline examples/basic-setup/src/my-command.js]: Resolved;

File: examples/colors/package.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "colors",
+  "version": "1.0.0",
```
Comment (ljharb) [inline examples/colors/package.json]: Does the linter require this line?

File: examples/form-validation/package.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "form-validation",
+  "version": "1.0.0",
```
Comment (ljharb) [inline examples/form-validation/package.json]: Or this one?

File: examples/profile-cards-graphql/src/main.js [hunk 6]
```diff
@@ -27,7 +28,8 @@ const QUERY = gql`
   }
 `;
 
-const props = ({ data }) => data.loading ? { users: [] } : { users: data.allProfiles };
+const props = ({ data }) =>
+  (data.loading ? { users: [] } : { users: data.allProfiles });
```
Comment (ljharb) [inline examples/profile-cards-graphql/src/main.js]: Was this line too long for the linter?

File: examples/colors/package.json (hunk 4 above)
Comment (sjy) [inline examples/colors/package.json]: nope, because some examples own the version field.

File: examples/form-validation/package.json (hunk 5 above)
Comment (sjy) [inline examples/form-validation/package.json]: not required from the existing eslint side

File: examples/profile-cards-graphql/src/main.js (hunk 6 above)
Comment (sjy) [inline examples/profile-cards-graphql/src/main.js]: The bracket matter, the linter said the expression is confusing;  It could be the same line;

Comment (ljharb) [inline examples/profile-cards-graphql/src/main.js]: Let's make it the same line then :-)

Comment (sjy) [inline examples/profile-cards-graphql/src/main.js]: Resolved :-p

**Comment by jongold** [comment]
Thanks @sjy!

**Pull request merged**